src/pages/Home/index.js

```diff
@@ -68,6 +70,8 @@ function generateInitial() {
   return initial;
 }
 
+const DEFAULT_EXTENSION = 'zip';
```

Comment:
This should be a member of `DEFAULT_PACK_META` in `configs.js`.

Reply:
Ok, will move.